client/src/pages/AgentHome/AgentHome.tsx

```diff
@@ -162,6 +165,7 @@ export default function AgentHome() {
           </>
```

Comment:
Not for this PR, but option for future dev, the items passed in renderItem could be cut into their own separate "dumb" components.